content/en/docs/reference/labels-annotations-taints.md

```diff
@@ -63,6 +63,16 @@ The Kubelet populates this label with the hostname. Note that the hostname can b
 This label is also used as part of the topology hierarchy.  See [topology.kubernetes.io/zone](#topologykubernetesiozone) for more information.
 
 
+## kubernetes.io/change-cause {#change-cause}
```

Comment:
At some future point we might mark this deprecated:
```markdown
## kubernetes.io/change-cause (deprecated) {#change-cause}
```
because of https://github.com/kubernetes/kubernetes/issues/40422